path.go

```diff
@@ -14,25 +17,21 @@ var prefixSize int
 
 // goPath is the deduced path based on the location of this file as compiled.
 var goPath string
+var srcDir string
 
 func init() {
-	_, file, _, ok := runtime.Caller(0)
-	if file == "?" {
-		return
-	}
-	if ok {
-		// We know that the end of the file should be:
-		// github.com/juju/errors/path.go
-		size := len(file)
-		suffix := len("github.com/juju/errors/path.go")
-		goPath = file[:size-suffix]
-		prefixSize = len(goPath)
+	goPath = getGOPATH()
+	srcDir = filepath.Join(goPath, "src")
+}
+
+func getGOPATH() string {
+	gopath := os.Getenv("GOPATH")
+	if gopath == "" {
```

Comment:
Does build.Default.GOPATH handle the environment piece?

Reply:
yeah, we should just relay on `build.Default.GOPATH` which is sufficient.